### ISSUE TITLE
Missing labels_ and predict function for K-Means

### ISSUE THREAD

**Issue by liufsd** [annoviko/pyclustering#515]
Great work. but k-means missing labels_ and predict  function like sklearn ~  https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html#sklearn.cluster.KMeans

**Comment by liufsd** [comment]
```
     Examples
    --------
    >>> from sklearn.cluster import KMeans
    >>> import numpy as np
    >>> X = np.array([[1, 2], [1, 4], [1, 0],
    ...               [10, 2], [10, 4], [10, 0]])
    >>> kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
    >>> kmeans.labels_
    array([1, 1, 1, 0, 0, 0], dtype=int32)
    >>> kmeans.predict([[0, 0], [12, 3]])
    array([1, 0], dtype=int32)
    >>> kmeans.cluster_centers_
    array([[10.,  2.],
           [ 1.,  2.]])

```

https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/cluster/k_means_.py

**Comment by annoviko** [comment]
Hello, @liufsd,
Thank you for your comments. About labels: pyclustering provides labels in different way, see **get_clusters()** method - https://pyclustering.github.io/docs/0.9.0/html/da/d22/classpyclustering_1_1cluster_1_1kmeans_1_1kmeans.html .

About prediction - I agree, it is useful feature, I will introduce this method.

**Comment by liufsd** [comment]
thanks～

**Comment by liufsd** [comment]
Hello , any progress ？

**Comment by annoviko** [comment]
Hello, @liufsd ,
Don't worry, it will be delivered in 0.9.1.

**Comment by liufsd** [comment]
ok~

**Comment by vivekverma080698** [comment]
Hi @annoviko how can I contribute to this predict feature ?. 
I need this feature in great hurry for OPTICS. Thank you

**Comment by annoviko** [comment]
Hello, @vivekverma080698 , at least article is required that described how to do that for OPTICS. And implementation itself is also can be contributed. In this thread we were talking about prediction for K-Means.

**Comment by liufsd** [comment]
@annoviko  Hello , any progress ？? ？
😢
